main.go

```diff
@@ -41,6 +40,11 @@ type device struct {
 }
 
 const appName = "NoiseTorch"
+var version = "unknown"
+var distribution = "custom"  // will be changed by build
+
+var updateURL = ""  // ditto
+var publicKeyString = ""  // ditto
```

Comment:
Please run `gofmt` to fix the formatting. And the "will be changed by build" comment should probably be moved up one line.